test/frontend/reducers/test_reducer_generator.js

```diff
@@ -0,0 +1,28 @@
+import { expect } from '../test_helper';
```

Comment:
same as the previous file , is this for something unrelated to programs and systems